src/main/java/io/numaproj/numaflow/batchmapper/DatumIterator.java

```diff
@@ -0,0 +1,20 @@
+package io.numaproj.numaflow.batchmapper;
+
+
+/**
+ * An iterator over a collection of {@link Datum} elements.
+ * Passed to {@link BatchMapper#processMessage(DatumIterator)} (DatumIterator)} method.
```

Comment:
Is the @link annotation working? I see one `{` and two `}`.

Reply:
done